roles/telemetry_autoscaling/tasks/test_autoscaling.yml

```diff
@@ -59,6 +59,19 @@
   register: busy_process
   with_items: "{{ vnf_instance_ip.stdout_lines }}"
 
+- name: Show queried metric
```

Comment:
As far as I know, this should work, since the update to install observabilityclient into the openstackclient is still there.

Reply:
Yes. Ideally we have all the new observabilityclient patches when running the role. This is taken care of by the playbook in ci/ upstream and by backports downstream. However this doesn't actually show anything. I'll submit another version, which actually displays the output of the command. I think this can be quite valuable for future debugging.